bigbluebutton-html5/public/locales/en.json

```diff
@@ -108,6 +108,12 @@
     "app.timer.hours": "hours",
     "app.timer.minutes": "minutes",
     "app.timer.seconds": "seconds",
+    "app.timer.abbreviation.hours": "h",
+    "app.timer.abbreviation.minutes": "min",
+    "app.timer.abbreviation.seconds": "sec",
+    "app.timer.preset.hours": "{hours}{abbr}",
```

Comment:
You just moved the interpolation to inside the react intl, avoid this case of use, implement the full string here or make changes in the one that are receiving it